src/settings/MappingProfiles/CheckboxGroupField/CheckboxGroupField.js

```diff
@@ -22,15 +26,19 @@ export const CheckboxGroupField = memo(({
             value={option.value}
             isEqual={isEqual}
             render={fieldProps => (
-              <Checkbox
-                {...fieldProps.input}
-                label={option.label}
-                innerClass={filtersLabelClass}
-                onChange={event => {
-                  onChange(event, option);
-                  fieldProps.input.onChange(event);
-                }}
-              />
+              <Row>
+                <Checkbox
+                  {...fieldProps.input}
+                  label={option.label}
+                  innerClass={filtersLabelClass}
+                  disabled={disabledFields[option.value]}
+                  onChange={event => {
+                    onChange(event, option);
+                    fieldProps.input.onChange(event);
+                  }}
+                />
+                {option.endAdornment}
```

Comment:
When naming it `endAdornment` I was thinking about more general name, so it would fit any other end element. Do you think it won't be used other that details?

Reply:
I meant the same. I think `details` is more informative and easier to reason about. `details` in my opinion is anything that is not placed in the label. If we understand that the name no longer fits, we will change it.